### PR TITLE
Fix cargo.toml version

### PR DESCRIPTION
**Pull request by derpmarine168**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,9 +11,9 @@ extern crate unicode_segmentation;
 use unicode_segmentation::UnicodeSegmentation;
 
 fn main() {
-    let s = "a̐éö̲\r\n";
```
Comment (kpozin) [inline README.md]: Please revert the sample string changes. Your editor seems to have applied NFC normalization to the file.

Comment (derpmarine168) [reply README.md]: oopsy - that's a issue w the github web editor.

**Comment by derpmarine168** [comment]
uh
github, why can't I change the branch?
🤦‍♂️
new PR I guess.

**Comment by derpmarine168** [comment]
<https://github.com/unicode-rs/unicode-segmentation/pull/48>

**Pull request closed without merging**